src/components/Footer.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import SnoqualmiePassLogo from "./UI/SnoqualmieLogo";
+import FacebookLogo from "./UI/FacebookLogo";
+import NSPLogo from "./UI/NSPLogo";
+
+const Footer = () => {
+  return (
+    <footer className="bg-zinc-800  w-full max-w-screen-xl mx-auto">
+      <hr className=" mx-auto border-gray-700" />
```

Comment:
See if you can manually format this file? Or maybe just resave to get the formatter to hit it again. There's some weird spacing bidness here that prettier should take care of (more than just nitpicking, seeing one spot where a formatter would have caught something is generally an indication that the entire file probably didn't get formatted, so just a general code smell)

Reply:
For some reason my prettier isn't formatting any CSS? I'll have to look into why its doing that. I've fixed it manually though!